src/Components/Categories/CategoriesFormCreate.js

```diff
@@ -0,0 +1,12 @@
+import React from 'react'
```

Comment:
this changes are not scope of this user story, please delete this file from this PR

Reply:
Done!